### ISSUE TITLE
Run integration tests against local product images

### ISSUE THREAD

**Issue by nightkr** [stackabletech/issues#520]
Currently we're good at running them against local _operators_ (just start tilt before starting the test), but running them against local product images requires you to modify all test definitions manually. It would be nice to have a shorthand for this.

**Comment by NickLarsenNZ** [comment]
Is the current method to edit the `tests/test-definitions.yaml` file? Eg:

```yml
dimensions:
  - name: opa
    values:
      - 0.0.0-dev # ⬅️
    # - 0.51.0
    # - 0.57.0
```

If so, would that become a feature request for beku.py? Eg: to `--set opa=0.0.0-dev` or something?
